Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -844,6 +845,22 @@ private void LoadModuleInFrame()
                 }
             }
         }
+        private void ResetTheme()
+        {
+            try
+            {
+                string ThemeName = Core.Managers.ThemeManager.GetCurrent(PortalSettings.Current.PortalId).Name;
+                string BaseEditorFolder = HttpContext.Current.Server.MapPath("~/Portals/" + PortalSettings.Current.PortalId + "/vThemes/" + ThemeName + "/editor");
+                string ThemeCss = HttpContext.Current.Server.MapPath("~/Portals/" + PortalSettings.Current.PortalId + "/vThemes/" + ThemeName + "/Theme.css");
+                if (!File.Exists(ThemeCss) && !Directory.Exists(BaseEditorFolder))
+                {
+                    ThemeManager.ProcessScss(PortalSettings.Current.PortalId, false);
+                    Config.Touch();
```

Comment:
Why Config.Touch()? This restarts site and should not be a requirement to create a new site.

Reply:
I have checked uxmode breaking when I will not do restart the application also tried with clear cache both server and client-side but not fixed with both